### PR TITLE
Implementation of 4 graph algorithms in python

### PR DESCRIPTION
**Pull request by ron-debajyoti**

### CODE REVIEW

File: code/graph-algorithms/eulerian_path/eulerian.py [hunk 1]
```diff
@@ -0,0 +1,118 @@
+# Python program to check if a given graph is Eulerian of not using eulerian path identification
+#Complexity : O(V+E)
```
Comment (arnavb) [inline code/graph-algorithms/eulerian_path/eulerian.py]: OpenGenus comment needed.

File: code/graph-algorithms/johnson_algorithm_shortest_path/johnsons_algo.py [hunk 2]
```diff
@@ -0,0 +1,149 @@
+
+
```
Comment (arnavb) [inline code/graph-algorithms/johnson_algorithm_shortest_path/johnsons_algo.py]: Same here.

File: code/graph-algorithms/left_view_binary_tree/left_view_binary.py [hunk 3]
```diff
@@ -0,0 +1,44 @@
+# A binary tree node
```
Comment (arnavb) [inline code/graph-algorithms/left_view_binary_tree/left_view_binary.py]: Same here

File: code/graph-algorithms/travelling_salesman_mst/travelling_salesman.py [hunk 4]
```diff
@@ -0,0 +1,15 @@
+
```
Comment (arnavb) [inline code/graph-algorithms/travelling_salesman_mst/travelling_salesman.py]: Here as well

File: code/graph-algorithms/eulerian_path/eulerian.py (hunk 1 above)
Comment (ron-debajyoti) [reply code/graph-algorithms/eulerian_path/eulerian.py]: My apologies but I didn't get u . Could u elaborate please ?

**Comment by arnavb** [comment]
Add the comment to each of your files:

```
# Part of Cosmos by OpenGenus Foundation
```

File: code/graph-algorithms/eulerian_path/eulerian.py (hunk 1 above)
Comment (yatharthshahjpr) [reply code/graph-algorithms/eulerian_path/eulerian.py]: Please add comment  /* Part of Cosmos by OpenGenus Foundation */ at the top of every code you contribute to this project  ! 👍

File: code/graph-algorithms/left_view_binary_tree/left_view_binary.py (hunk 3 above)
Comment (yatharthshahjpr) [reply code/graph-algorithms/left_view_binary_tree/left_view_binary.py]: Please add comment  /* Part of Cosmos by OpenGenus Foundation */ at the top of every code you contribute to this project  ! 👍

File: code/graph-algorithms/eulerian_path/eulerian.py [hunk 5]
```diff
@@ -1,3 +1,5 @@
+# Part of Cosmos by OpenGenus Foundation
```
Comment (yatharthshahjpr) [inline code/graph-algorithms/eulerian_path/eulerian.py]: yes this is fine ! 👍

Comment (ron-debajyoti) [reply code/graph-algorithms/eulerian_path/eulerian.py]: I have added the comments.

Comment (ron-debajyoti) [reply code/graph-algorithms/eulerian_path/eulerian.py]: @AdiChat  please get this PR reviewed #1682

**Comment by AdiChat** [comment]
Looks great 😍 

Hope you are enjoying your journey 🚆  with **OpenGenus**

**Pull request merged**